Lib/fontTools/varLib/instancer.py

```diff
@@ -132,132 +130,152 @@ def instantiateCvar(varfont, location):
         del varfont["cvar"]
 
 
-def setMvarDeltas(varfont, location):
+def setMvarDeltas(varfont, deltaArray):
     log.info("Setting MVAR deltas")
 
     mvar = varfont["MVAR"].table
-    fvar = varfont["fvar"]
-    varStoreInstancer = VarStoreInstancer(mvar.VarStore, fvar.axes, location)
     records = mvar.ValueRecord
-    # accumulate applicable deltas as floats and only round at the end
```

Comment:
turns out it's actually not necessary to accumulate and round at the end in this case, because each MVAR record can only refer to one item in the VarStore at a time.